winmake.ps1

```diff
@@ -219,9 +227,7 @@ function Build-Ginkgo{
         return
     }
     Write-Host "Building Ginkgo"
-    Push-Location ./test/tools
-    Run-Command "go build -o build/ginkgo.exe ./vendor/github.com/onsi/ginkgo/v2/ginkgo"
-    Pop-Location
+    Run-Command "go build -o ./test/tools/build/ginkgo.exe ./vendor/github.com/onsi/ginkgo/v2/ginkgo"
```

Comment:
@l0rd with this change, ginkgo is being built from the main repo vendor, while it was using test/tools repo vendor before.

Reply:
Yes. I did that to avoid a "versions don't match" warning at every execution. But that may have been a mistake because I don't know why have ginkgo declared twice (and with two non matching versions). I am happy to fix it if there is a better way to solve the problem.

Reply:
Thanks for the explanation. This is being fixed in #25732.